Shared/Samples/Query feature count and extent/README.md

```diff
@@ -0,0 +1,47 @@
+# Query feature count and extent
+
+Zoom to features matching a query and count the features in the current visible extent.
+
+![Image of Query feature count and extent sample](query-feature-count-and-extent.png)
+
+## Use case
+
+Queries can be used to search for features in a feature table using text entry. This is helpful for finding a specific feature by name in a large feature table. A query can also be used to count features in an extent. This could be used to count the number of traffic incidents in a particular region when working with an incident dataset for a larger area.
+
+## How to use the sample
+
+Use the button select a state, then use the picker to zoom to the extent of the state specified. Use the button to count the features in the current extent.
```

Comment:
```suggestion
Use the button Select State, then use the picker to zoom to the extent of the state specified. Use the button to count the features in the current extent.
```
I think capitalizing "Select State" makes it clearer that it is referring to the button as it is in the UI.

Reply:
I appreciate the feedback; thank you!